cumulusci.yml

```diff
@@ -87,6 +98,13 @@ flows:
                     - when: "'npsp' not in tasks.get_installed_packages()"
                       action: hide
                 when: "org_config.has_minimum_package_version('npsp', '1.0')"
+            4:
+                task: site_template:install_managed_beta
```

Comment:
@seanpat09 , this is temporary?   Can we create a work item to change this to "install production"?